pkgs/development/python-modules/benqprojector/default.nix

```diff
@@ -0,0 +1,43 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  hatchling,
+  hatch-vcs,
+  pyserial,
+  pyserial-asyncio-fast,
+}:
+
+buildPythonPackage rec {
+  pname = "benqprojector";
+  version = "0.1.6";
+  pyproject = true;
+
+  src = fetchPypi {
```

Comment:
`fetchFromGitHub` is preferred

Reply:
@dotlambda For what reason, performance? If that's a general rule, it could be mentioned here: https://nixos.org/manual/nixpkgs/stable/#contributing-guidelines

Reply:
E.g. because of the xz disaster.

Reply:
Got your point, it's indeed problematic that source distributions and wheels differ from the source repo…